test/ReactSetStateTestComponent2.js

```diff
@@ -79,6 +79,14 @@ class ReactSetStateTestComponent2 extends React.Component {
         this.setState(this.innerTransactionalSetStateCallback.bind(this), this.innerSetStateCallback.bind(this));
     }
 
+    componentDidCatch(_, __) {
+        this.recordLifecycleCall('componentDidCatch');
```

Comment:
Should we record the value of the two arguments here to verify they are being passed along as expected?